pkg/client/unversioned/fake/fake.go

```diff
@@ -71,15 +73,23 @@ func (c *RESTClient) Delete() *restclient.Request {
 
 func (c *RESTClient) request(verb string) *restclient.Request {
 	config := restclient.ContentConfig{
-		ContentType:  runtime.ContentTypeJSON,
-		GroupVersion: testapi.Default.GroupVersion(),
-		Codec:        c.Codec,
+		ContentType:          runtime.ContentTypeJSON,
+		GroupVersion:         testapi.Default.GroupVersion(),
+		NegotiatedSerializer: c.NegotiatedSerializer,
+	}
+	ns := c.NegotiatedSerializer
```

Comment:
This is just fake client for tests. I agree it would be bad in real code, but this is just testing thing...